gui-apps/waybar/metadata.xml

```diff
@@ -21,7 +21,6 @@
 		<flag name="mpd">Enable support for the Music Player Daemon</flag>
 		<flag name="mpris">Enable support for mpris</flag>
 		<flag name="network">Enable libnl support for network related features</flag>
-		<flag name="sndio">Enable support for volume control via sndio</flag>
```

Comment:
If this is just for volume control, then this description is *much* better suited than the generic description.